### PR TITLE
More comprehensive setup script

### PR DESCRIPTION
**Pull request by enjeyw**
Fixes the issue with the python setup script and creates new more comprehensive bash one:
`quick_setup_script.sh`

To use:
- set env variable `MASTER_WALLET_PK` to the master private key found in your local ini secrets file 

-run `quick_setup_script.sh [activation path for your python env] `

- once that's finished run, start app and worker as normal. Ganache will already be running

### CODE REVIEW

File: ganache_cli.sh [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
 
-ganache-cli -l 80000000 -i 42 --db './ganacheDB' \
---account="0x00f49bc12b2e102e072ee086482e700ad7fe2b5ee417697b13ca04e4dc1572d9,10000000000000000000000000"
\ No newline at end of file
+ganache-cli -l 80000000 -i 42 \
+--account="0x00f49bc12b2e102e072ee086482e700ad7fe2b5ee417697b13ca04e4dc1572d9,10000000000000000000000000" \
+--db './ganacheDB'
```
Comment (michielderoos) [inline ganache_cli.sh]: When running this, I get `Error: ENOENT: no such file or directory, open 'ganacheDB/!blockHashes!0x8b02e4a1d0666e20b88b5fabce4650a62353991694cc5c1e28b340f022ef5d2c'`

Comment (enjeyw) [reply ganache_cli.sh:5]: hmmm probably because the current script isn't designed to run with persistence. lemme make them both compatible.

Comment (enjeyw) [reply ganache_cli.sh:8]: ok should be fixed now @michielderoos: if you launch with the quick setup script, shut down and then start up with this script, everything should behave,  though the private key does need to be sent in the env variable

**Comment by tristanhcole** [comment]
This PR should include an update to the Readme file

**Comment by tristanhcole** [comment]
```
[2020-04-02 11:05:00,062: ERROR/MainProcess] Task eth_manager.celery_tasks._process_deploy_contract_transaction[4ef9d227-d434-474d-8258-4484b54ea805] raised unexpected: PreBlockchainError('Transaction 3: {\'message\': "sender doesn\'t have enough funds to send tx. The upfront cost is: 3041116000000000 and the sender\'s account only has: 0", \'code\': -32000, \'data\': {\'stack\': "Error: sender doesn\'t have enough funds to send tx. The upfront cost is: 3041116000000000 and the sender\'s account only has: 0\\n    at /usr/local/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:61:655260\\n    at /usr/local/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:37:1762210\\n    at /usr/local/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:37:1762034\\n    at /usr/local/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:25:135768\\n    at /usr/local/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:25:139529\\n    at Object.return (/usr/local/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:25:139106)\\n    at /usr/local/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:25:137358\\n    at e (/usr/local/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:25:139477)\\n    at /usr/local/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:25:139488\\n    at /usr/local/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:25:136570\\n    at process._tickCallback (internal/process/next_tick.js:61:11)", \'name\': \'Error\'}}',)
Traceback (most recent call last):
  File "/Users/tristancole/VirtualEnvs/SempoBlockchain_env/lib/python3.6/site-packages/celery/app/trace.py", line 385, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/Users/tristancole/VirtualEnvs/SempoBlockchain_env/lib/python3.6/site-packages/celery/app/trace.py", line 650, in __protected_call__
    return self.run(*args, **kwargs)
  File "/Users/tristancole/VirtualEnvs/SempoBlockchain_env/lib/python3.6/site-packages/sentry_sdk/integrations/celery.py", line 161, in _inner
    reraise(*exc_info)
  File "/Users/tristancole/VirtualEnvs/SempoBlockchain_env/lib/python3.6/site-packages/sentry_sdk/_compat.py", line 57, in reraise
    raise value
  File "/Users/tristancole/VirtualEnvs/SempoBlockchain_env/lib/python3.6/site-packages/sentry_sdk/integrations/celery.py", line 156, in _inner
    return f(*args, **kwargs)
  File "/Users/tristancole/VirtualEnvs/SempoBlockchain_env/lib/python3.6/site-packages/celery/app/base.py", line 474, in run
    return task._orig_run(*args, **kwargs)
  File "/Users/tristancole/PycharmProjects/SempoBlockchain/eth_worker/eth_manager/celery_tasks.py", line 188, in _process_deploy_contract_transaction
    args, kwargs, gas_limit, task_id)
  File "/Users/tristancole/PycharmProjects/SempoBlockchain/eth_worker/eth_manager/processor.py", line 129, in process_deploy_contract_transaction
    return self.process_transaction(transaction_id, constructor, gas_limit=gas_limit)
  File "/Users/tristancole/PycharmProjects/SempoBlockchain/eth_worker/eth_manager/processor.py", line 216, in process_transaction
    raise e
  File "/Users/tristancole/PycharmProjects/SempoBlockchain/eth_worker/eth_manager/processor.py", line 190, in process_transaction
    raise PreBlockchainError(message, True)
eth_manager.exceptions.PreBlockchainError: Transaction 3: {'message': "sender doesn't have enough funds to send tx. The upfront cost is: 3041116000000000 and the sender's account only has: 0", 'code': -32000, 'data': {'stack': "Error: sender doesn't have enough funds to send tx. The upfront cost is: 3041116000000000 and the sender's account only has: 0\n    at /usr/local/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:61:655260\n    at /usr/local/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:37:1762210\n    at /usr/local/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:37:1762034\n    at /usr/local/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:25:135768\n    at /usr/local/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:25:139529\n    at Object.return (/usr/local/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:25:139106)\n    at /usr/local/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:25:137358\n    at e (/usr/local/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:25:139477)\n    at /usr/local/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:25:139488\n    at /usr/local/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:25:136570\n    at process._tickCallback (internal/process/next_tick.js:61:11)", 'name': 'Error'}}
[2020-04-02 11:05:00,426: INFO/MainProcess] Received task: eth_manager.celery_tasks.get_task[ba331a88-0363-46b9-8b7d-647de6004fe8]  
[2020-04-02 11:05:00,434: INFO/MainProcess] Task eth_manager.celery_tasks.get_task[ba331a88-0363-46b9-8b7d-647de6004fe8] succeeded in 0.0068985040002189635s: {'id': 2, 'status': 'FAILED', 'prior_tasks': [], 'posterior_tasks': [], 'transactions': [3]}

```

I'm still getting an error running this script from fresh

File: quick_setup_script.sh [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  mkdir ganacheDB
+  ganache-cli -l 80000000 -i 42 --account="${MASTER_WALLET_PK},10000000000000000000000000" --db './ganacheDB' &
+fi
+
+sleep 5
+
+set -e
+
+echo ~~~Starting worker
+celery -A eth_manager worker --loglevel=INFO --concurrency=8 --pool=eventlet -Q processor,celery &
```
Comment (tristanhcole) [inline quick_setup_script.sh:60]: this threw an error for me

```
Error: 
Unable to load celery application.
The module eth_manager was not found.

```

File: quick_setup_script.sh [hunk 3]
```diff
@@ -12,6 +12,14 @@ echo aborting
...
 fi
 
+if [ -z ${MASTER_WALLET_PK+x} ]
+then
+echo "\$MASTER_WALLET_PK is empty"
+exit 0
+else
+echo "\$MASTER_WALLET_PK is NOT empty"
+fi
+
```
Comment (tristanhcole) [inline quick_setup_script.sh:17]: pretty sure [this](https://github.com/teamsempo/SempoBlockchain/pull/202#issuecomment-607548767) was caused by MASTER_WALLET_PK being undefined... so added check

File: README.md [hunk 4]
```diff
@@ -33,9 +33,18 @@ cd app
...
 ```
 
-### Enable the Simulator (Optional)
+### Setup Script
+
+For quick setup, run `quick_setup_script.sh` with `MASTER_WALLET_PK` set as an environment variable to the master private key found in `/config_files/secret/local_secrets.ini/`.
+
+```
+quick_setup_script.sh [activation path for your python env]
+```
```
Comment (tristanhcole) [inline README.md:42]: This kinda makes the ###Blockchain section redundant. Probably need to rewrite the readme

**Comment by enjeyw** [comment]
> Works well for me now!
> 
> Could be good to update the setup script to create multiple organisations and some fake data for better testing @enjeyw

Fake data is a different script IMO, but agree we should have it!

File: quick_setup_script.sh (hunk 3 above)
Comment (enjeyw) [reply quick_setup_script.sh:17]: oh nice!

File: README.md (hunk 4 above)
Comment (enjeyw) [reply README.md:42]: I'll tweak it - should only be a small change.

File: contract_setup_script.py [hunk 5]
```diff
@@ -278,53 +278,35 @@ def _base_setup(s, reserve_token_id):
...
 
+    bind_1 = s.bind_me_to_organisation_as_admin(org_id)
 
 def local_setup():
     s = Setup(
         api_host='http://0.0.0.0:9000/api/v1/',
-        email=os.environ.get('LOCAL_EMAIL'),
-        password=os.environ.get('LOCAL_PASSWORD')
+        email=os.environ.get('LOCAL_EMAIL', 'admin@acme.org'),
+        password=os.environ.get('LOCAL_PASSWORD', 'C0rrectH0rse')
```
Comment (michielderoos) [inline contract_setup_script.py:299]: Battery Staple

**Pull request merged**